src/utils/flags/useEnableKubevirtMenuFlags.ts

```diff
@@ -0,0 +1,24 @@
+import VirtualMachineClusterInstancetypeModel from '@kubevirt-ui/kubevirt-api/console/models/VirtualMachineClusterInstancetypeModel';
+import VirtualMachineClusterPreferenceModel from '@kubevirt-ui/kubevirt-api/console/models/VirtualMachineClusterPreferenceModel';
+import { K8sVerb, SetFeatureFlag, useAccessReview } from '@openshift-console/dynamic-plugin-sdk';
+
+import { FLAG_KUBEVIRT_INSTANCETYPES, FLAG_KUBEVIRT_PREFERENCES } from './consts';
+
+const useKubevirtMenuFlags = (setFeatureFlag: SetFeatureFlag) => {
```

Comment:
nitpick: Why are the file name and hook name different?

Reply:
U are very right!

Reply:
done